skranger/ensemble/regressor.py

```diff
@@ -310,6 +310,8 @@ def predict_quantiles(self, X, quantiles):
 
         forest = self._get_terminal_node_forest(X)
         terminal_nodes = np.array(forest["predictions"]).astype(int)
+        if len(terminal_nodes.shape) < 2:
+            terminal_nodes = np.reshape(terminal_nodes, (1, -1))
```

Comment:
```suggestion
        terminal_nodes = np.atleast_2d(terminal_nodes)
```
I think we can accomplish the same thing here with this, which is a bit simpler.

Reply:
Looks even better! Thanks for reviewing this.

Reply:
Sure thing. Can you get rid of the if statement on the line before? After that I think we can merge.

Reply:
Got it. I removed the unneeded if statement.